E2E/pageObjects/game.page.ts

```diff
@@ -133,6 +143,21 @@ export = {
     game.voteIsVisible(chessPiece, tile, value);
   },
 
+  async openPopUp (row: number, column: number) {
+    I.usePlaywrightTo('open pop-up', async ({ page }) => {
+      await page.evaluate(`document.querySelector('div[data-testid = "chess-tile-${row}-${column}"] > div.pop-over').classList.add("pop-over-opened")`);
```

Comment:
We have rows and columns numbers defined in test_data/chessTile.ts file. Maybe those can be reused here?